workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -913,25 +909,21 @@ def sync_quant_files(output_path, samples: List[Sample]):
 
     page_size = 100
     # split the samples in groups and download each one individually
-    pool = multiprocessing.Pool(processes=MULTIPROCESSING_WORKER_COUNT)
-
-    # for each sample we need it's latest quant.sf file we don't want to query the db
-    # for all of them, so we do it in groups of 100, and then download all of the computed_files
-    # in parallel
-    for sample_page in (samples[i*page_size:i+page_size] for i in range(0, len(samples), page_size)):
-        sample_and_computed_files = []
-        for sample in sample_page:
-            latest_computed_file = sample.get_most_recent_quant_sf_file()
-            if not latest_computed_file:
-                continue
-            output_file_path = output_path + sample.accession_code + "_quant.sf"
-            sample_and_computed_files.append((latest_computed_file, output_file_path))
-
-        # download this set of files, this will take a few seconds that should also help the db recover
-        pool.map(downlad_computed_file, sample_and_computed_files)
-        num_samples += len(sample_and_computed_files)
+    with ThreadPoolExecutor(max_workers=16) as executor:
```

Comment:
Can you move this number back to the top, define it as a constant, and then leave a comment explaining why you picked 16?

Reply:
good idea, 16 seemed reasonable but not sure if it's the best number for this. Do you have any suggestions on how to calculate a better number for this?

Reply:
Hmm, the way it was being calculated based on the number of cores of the instance seemed like a good way to do it to me. Not sure why we'd wanna switch away from that. Whether we're using processes or threads we should still base the number off the number of threads.

Reply:
just updated it and used: `max(1, math.floor(multiprocessing.cpu_count()/2) - 1)`